samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/evaluation/QuestionScoresBean.java

```diff
@@ -495,11 +496,11 @@ public void setSelectedSectionFilterValue(String param ) {
       }
   }
 
-public List getAllAgents()
+public List getAllAgents(boolean valueChanged)
 {
 	  String publishedId = ContextUtil.lookupParam("publishedId");
 	  QuestionScoreListener questionScoreListener = new QuestionScoreListener();
-	  if (!questionScoreListener.questionScores(publishedId, this, false))
+	  if (!questionScoreListener.questionScores(publishedId, this, valueChanged))
```

Comment:
@bgarciaentornos your change seems fine. but i think all you are doing is flipping this from false to true? 

there is no need to pass a value, is there?

Reply:
It will only be true when the value is changed within the 'resetCache if' I entered for S2U-5, false as before for the first load. If you see my changes against yours you will spot it, if you look at the final changes you might not find it.

Reply:
ahhhh yes, thanks for explaining. i understand now.